packaging/test_irods_resource_plugin_s3.py

```diff
@@ -150,6 +150,9 @@ def test_put_get_small_file_in_repl_node(self):
     def test_put_get_large_file_in_repl_node(self):
         pass
 
+    @unittest.skipIf(True, 'test does not work in decoupled because we are using same bucket for multiple resources')
```

Comment:
I think you can refactor this to just use `.skip("your message here ...")`.

Same for the others.

Reply:
I made the changes to the newly inserted lines.  I left the unchanged lines alone.